subprojects/migration/src/main/java/org/mockito/migration/MockitoV1CompatibilityMatcher.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2017 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package org.mockito.migration;
+
+import java.lang.reflect.Method;
+import org.mockito.ArgumentMatcher;
+import org.mockito.Incubating;
+
+/**
+ * Base class for classes that implement {@link ArgumentMatcher}.
```

Comment:
Fantastic javadoc. Thank you!

Can you add information that folks need to depend on new artifact, e.g. 'mockito-migration'?